code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -174,6 +174,50 @@
 	body_parts_covered = CHEST|ARMS|GROIN|LEGS|FEET
 	flags_inv = HIDESHOES|HIDEJUMPSUIT
 
+//I will ensure that wearing this will warent a sufficient beating to take it off again
+/obj/item/clothing/head/chungus
+	name = "chungus hat"
+	desc = "BIG CHUNGUS HUER DU UUUUUUUUUUUUUUUU CHUNGUS FUNNY REDDIT MOMENT BIG GROBE CHUNGUS KIONO REEVES WHOLE SOME 100 ELON MUSK."
+	icon_state = "bigchungushead"
+	armor = list("melee" = -80, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 0, "fire" = 0, "acid" = 100 ,"stamina" = 0)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+	clothing_flags = SNUG_FIT
+
+/obj/item/clothing/suit/chungus
+	name = "chungus suit"
+	desc = "BIG CHUNGUS HUER DU UUUUUUUUUUUUUUUU CHUNGUS FUNNY REDDIT MOMENT BIG GROBE CHUNGUS KIONO REEVES WHOLE SOME 100 ELON MUSK."
+	icon_state = "chungus"
+	armor = list("melee" = -80, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 0, "fire" = 0, "acid" = 100 ,"stamina" = 0)
+	body_parts_covered = CHEST|ARMS|GROIN|LEGS|FEET
+	flags_inv = HIDESHOES|HIDEJUMPSUIT
+	var/list/datum/brain_trauma/chunguslist = list()
+	var/list/datum/brain_trauma/storedtraumas = list()
+
+/obj/item/clothing/suit/chungus/Initialize()
+	. = ..()
+	chunguslist += /datum/brain_trauma/severe/discoordination
+	chunguslist += /datum/brain_trauma/mild/stuttering
+	chunguslist += /datum/brain_trauma/mild/dumbness
+	chunguslist += /datum/brain_trauma/mild/speech_impediment
+	chunguslist += /datum/brain_trauma/mild/expressive_aphasia
+
+/obj/item/clothing/suit/chungus/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(slot == ITEM_SLOT_OCLOTHING)
```

Comment:
That both doesn't work and is not used anywhere when doing clothing checking.  iam guesing these bits are mutaly exclusive for some reason.

Reply:
It does work with the method you described but my could would be inconsistent with other clothing that apply effects on wear

Reply:
To copypaste my response in discord for anyone else reading:

_currently* there is no real benefit or downside to clothing slot bit operators over equality in all current implementations, this was mostly done for consistency as far as I can tell
However, once all the checks are brought in line, we should be able to make clothing that can be equipped in multiple different slots. 

The bitflag change was recent so that's why most clothing checks aren't up to date, the original author missed those